pyrgg/params.py

```diff
@@ -30,15 +30,15 @@
 }
 
 MENU_ITEMS2 = {
-    "vertices": "- Vertices Number : ",
-    "max_weight": "- Max Weight : ",
-    "min_weight": "- Min Weight : ",
-    "min_edge": "- Min Edge Number : ",
-    "max_edge": "- Max Edge Number : ",
-    "sign": "- Unsigned[0] or Signed[1]",
-    "direct": "- Undirected[0] or Directed[1]",
-    "self_loop": "- No Self Loop[0] or Self Loop[1]",
-    "multigraph": "- Simple[0] or Multigraph[1]",
+    1: ["vertices", "- Vertices Number : "],
+    2: ["min_edge", "- Min Edge Number (Connected to Each Vertex) : "],
+    3: ["max_edge", "- Max Edge Number (Connected to Each Vertex) : "],
+    4: ["min_weight", "- Min Weight : "],
+    5: ["max_weight", "- Max Weight : "],
+    6: ["sign", "- Unsigned[0] or Signed[1]"],
+    7: ["direct", "- Undirected[0] or Directed[1]"],
+    8: ["self_loop", "- No Self Loop[0] or Self Loop[1]"],
+    9: ["multigraph", "- Simple[0] or Multigraph[1]"]
```

Comment:
Why is this a better idea than the former approach?

Reply:
In order to have this ability to control menu items order.
It wasn't possible in the previous versions.

**Previous version:**

![image](https://user-images.githubusercontent.com/7515099/186122535-add30503-94dc-4504-a7c1-149d72fb8d0d.png)


**New version:**
![image](https://user-images.githubusercontent.com/7515099/186122170-17f69a14-a159-4dc3-94ae-418bb794c737.png)